lib/wanda_web/schemas/catalog/check.ex

```diff
@@ -89,6 +89,11 @@ defmodule WandaWeb.Schemas.CatalogResponse.Check do
             expression: %Schema{
               type: :string,
               description: "Expression to be evaluated to get the expectation result"
+            },
+            failure_message: %Schema{
+              type: :string,
+              nullable: true,
+              description: "Expression to be evaluated in case of failure"
```

Comment:
I would change the `description` to:
`Advanced failure message expression to return if the check fails` or similar.
At the to emphasize that this is something the user will get back